### PR TITLE
worker: allow destruction of individual workers

### PR DESCRIPTION
**Pull request by melvinw**
This may also be useful.

### CODE REVIEW

File: protobuf/service.proto [hunk 1]
```diff
@@ -13,6 +13,7 @@ service BESSControl {
   rpc ResetWorkers (EmptyRequest) returns (EmptyResponse) {}
   rpc ListWorkers (EmptyRequest) returns (ListWorkersResponse) {}
   rpc AddWorker (AddWorkerRequest) returns (EmptyResponse) {}
+  rpc DestroyWorker (DestroyWorkerRequest) returns (EmptyResponse) {}
   // TODO: delete_worker()
```
Comment (sangjinhan) [inline protobuf/service.proto]: TODO is done :-)

Comment (melvinw) [inline protobuf/service.proto]: Fixed

**Comment by sangjinhan** [comment]
```
$ run samples/multicore
$ delete worker 3              # No error message
$ delete worker 128          # Incorrect error message
*** Error: Missing 'wid' field
  BESS daemon response - errno=22 (EINVAL: Invalid argument)
$ delete worker 1              # Should not success since something is running on the worker. Module src1 becomes lost
```

**Comment by sangjinhan** [comment]
`add worker` command would be useful too :)

(I promise two cats)

**Comment by melvinw** [comment]
All fixed up

**Comment by sangjinhan** [comment]
Merged, thanks!
tumblr_m7vn84gwqd1qa0rxqo1_1280

**Pull request merged**